graylog2-web-interface/src/components/graylog/Icon.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import 'font-awesome/css/font-awesome.css';
+
+const Icon = ({ ...props }) => {
```

Comment:
What about passing a `type` prop, that specifies the name of the icon? For example for `<Icon type="times" />` it generates `<i className="fa fa-times" />`. This way we could also generate a mapping translating from font awesome to something else, if we move away from it.

Reply:
That's the intention, just going through one step at a time for clearer commit history for easier review 😄